jenkins/createrepo.sh

```diff
@@ -0,0 +1,16 @@
+# adapted from QE-RPMs jobs (thanks jsefler)
```

Comment:
I cannot see that this file is used in `Jenkinsfile`. I cannot recall that we have ever created rpm repository from artifacts. Can it be useful for QE?

Reply:
It was in the old code so I added it here.  I did not add it to the Jenkinsfile because it doesn't actually work.  I also don't see that the repo was used anywhere.  QE creates our own RPMs so we don't need it.  If you don't use it, I vote to delete this file.

Reply:
OK, let's delete it.

Reply:
done!